ci/dhis2-verify-app.yml

```diff
@@ -69,11 +69,11 @@ jobs:
     #e2e:
     #    runs-on: ubuntu-latest
     #    if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
-    #    needs: [cypress_id]
     #
     #    strategy:
+    #        fail-fast: false
```

Comment:
Necessary since cypress needs all parallel runs to finish. Otherwise the task might hang.

Reply:
👍